proposals/lip-0063.md

```diff
@@ -49,19 +49,30 @@ The following table defines some constants that will be used in the remainder of
    </td>
    <td>byte string of length 96, each byte set to zero
    </td>
-   <td>A dummy value for a <a href="https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md#public-key-registration-and-proof-of-possession">proof of possession</a>. The PoS module requires that each validator account contained in a snapshot/genesis block has such an entry. However, the proofs of possession are not verified for this particular snapshot block.
+   <td>A dummy value for a . The PoS module requires that each validator account contained in a snapshot/genesis block has such an entry. However, the proofs of possession are not verified for this particular snapshot block.
```

Comment:
The sentence is incomplete now. Maybe you want to say 'A dummy value for a proof of possession'? Not sure why the link was removed.

Reply:
This was a mistake, I reverted back to the old version.

These html tables a annoying.